config/environments/development.rb

```diff
@@ -12,6 +12,22 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Do not care if the mailer cannot send.
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.perform_caching = false
+  config.action_mailer.delivery_method = :notify
+  config.action_mailer.notify_settings = {
+    api_key: ENV.fetch("GOVUK_NOTIFY_API_KEY"),
```

Comment:
We probably don't want to actually be sending emails in dev and testing. It just causes too many mishaps. There are many good solutions for this one of them being: https://github.com/ryanb/letter_opener . It just opens the email in the browser on localhost which is useful for debugging and eliminates the risk of sending emails from dev to actual users.

Reply:
When you create the API key for notify, you can create a dev key, which will not send any emails

Reply:
👍

Reply:
Yep, like Ethan said Notify is pretty good at managing it, and I do like actually sending emails, makes me feel safer that it will actually work in production :)